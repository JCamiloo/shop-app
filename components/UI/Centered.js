import React from "react";
import { View, StyleSheet } from "react-native";

const Centered = (props) => {
  return <View style={styles.container}>{props.children}</View>;
};

export default Centered;

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: "center",
    justifyContent: "center",
  },
});
