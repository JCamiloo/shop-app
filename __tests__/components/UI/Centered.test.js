import React from "react";
import Centered from "../../../components/UI/Centered";
import renderer from "react-test-renderer";

describe("Centered component tests", () => {
  test("Centered component renders correctly", () => {
    const tree = renderer.create(<Centered />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
