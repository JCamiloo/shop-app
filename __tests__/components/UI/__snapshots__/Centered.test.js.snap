// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Centered component tests Centered component renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
/>
`;
